lib/internal/validators.js

```diff
@@ -304,8 +304,10 @@ const validateArray = hideStackFrames((value, name, minLength = 0) => {
 /** @type {validateStringArray} */
 function validateStringArray(value, name) {
   validateArray(value, name);
-  for (let i = 0; i < value.length; i++) {
-    validateString(value[i], `${name}[${i}]`);
+  for (let i = 0; i < value.length; ++i) {
+    if (typeof value[i] !== 'string') {
```

Comment:
Here and below it would probably be a good idea to add a comment in case someone in the future tries to revert back to `validate*()`.

Reply:
done